packages/paste-website/src/pages/primitives/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Primitives overview
+description: Our primitives are rad
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {ComponentOverviewTable} from '../../components/component-overview-table';
+import {SidebarCategoryRoutes} from '../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+
+# Primitives Overview
+
+<Callout>
+  <CalloutTitle as="h4">Don't see a primitive, utility, or component you need listed here?</CalloutTitle>
+  <UnorderedList>
+    <ListItem>
+      First off, we recommend{' '}
+      <Anchor href="https://github.com/twilio-labs/paste/issues">creating an issue on GitHub</Anchor> with your
+      component, primitive, or utility proposal.
+    </ListItem>
+    <ListItem>
+      While not all requests are accepted for the design system, we may be able to suggest suitable alternatives - for
+      this, drop us a note on #help-design-system or join our Weekly Office hours.
+    </ListItem>
+    <ListItem>
+      We've also written a guide for{' '}
+      <Anchor href="/tokens/how-to-compose-custom-ui-with-tokens">
+        how you can build custom components with Tokens
+      </Anchor>{' '}
+      and still benefit from Paste design updates
+    </ListItem>
+  </UnorderedList>
+</Callout>
+<ComponentOverviewTable
+  categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
+  componentsList={props.data.allPastePrimitive.edges}
+/>
+
+export const pageQuery = graphql`
+  {
+    allPastePrimitive {
+      edges {
+        node {
+          version
+          name
```

Comment:
Calling these "Primitive" in a page titled primitive is very redundant. Can we call these "Unstyled" <Component Name>

Reply:
Are you referencing the "primitive accordion", "primitive alert", etc?

Reply:
This is based on the package name at the moment. It might get awkward to try and trim the package name of "primitive" 🤔 

Is it a huge deal

Reply:
It's extremely redundant and not easy to read.

Reply:
good for now